guides/android-setup.md

```diff
@@ -48,58 +47,91 @@ A quick way to start is by adding your App ID to the `application` section of th
+{% endnotewarning %}
+
+Open the TelemetryDeck Dashboard, navigate to "Explore > Recent Signals" and make sure "Test Mode" is enabled. You should see automatic signals appear after launching your app.
+
+---
 
-By default, the TelemetryDeck SDK will automatically track user activity based on when the app starts, or when it returns from the background. This means, that this is all you need to do to start sending signals to TelemetryDeck.
+{% noteinfo "Ready for Basic Insights" %}
+Congratulations! With just the SDK integration you've completed, TelemetryDeck will automatically track user sessions, app launches, and device information. This basic setup provides valuable built-in insights without any additional code.
 
-You can now go to the [Dashboard](https://dashboard.telemetrydeck.com/) and watch your signals pour in, for example in the "Recent Signals" list. Signals produced while working in your IDE will automatically be tagged as **Test Signals** so remember to enable Test Mode in the Dashboard to see your testing data your Insights and charts. Disable Test Mode in the dashboard to see your production data once your app is live.
+You can now build and release your app. Once users start using it, your TelemetryDeck dashboard will begin showing data about user behavior, device types, and other key metrics.
+{% endnoteinfo %}
 
 ## Privacy Policy and Opt-Out
 
 You don't need to update your privacy policy, [but we recommend you do it anyway](/docs/guides/privacy-faq/#do-i-need-to-add-telemetrydeck-to-my-privacy-policy%3F).
 
+## Enhancing Your Analytics (Optional)
 
-## Advanced: Sending Signals Manually
+While basic session tracking provides valuable information, sending custom signals lets you answer questions specific to how users engage with *your* app.
 
-Let's send a signal manually. Navigate to a Kotlin file and add the following code at the top:
+### Sending Custom Signals
+
+Navigate to a Kotlin file and add the following code at the top:
 
 ```kotlin
 import com.telemetrydeck.sdk.TelemetryDeck
 ```
 
-Then, in an appropriate function, you can send your manual first signal.
-
-By default, you're not sending signals immediately; you're _enqueing_ them. This batches them up to be sent every now and then, and is the recommended way to send signals, as it will conserve a lot of battery life for your users. To enqueue a signal to be sent by TelemetryDeck use this line:
+Then, in an appropriate function, you can send your first custom signal:
 
 ```kotlin
 TelemetryDeck.signal("pizzaOrderConfirmed")
 ```
 
-Of course you can replace `somethingSpecificHappened` by any other name you'd like.
+By default, you're not sending signals immediately; you're _enqueing_ them. This batches them up to be sent every now and then, and is the recommended way to send signals, as it will conserve a lot of battery life for your users.
```

Comment:
This section was removed from the SwiftSDK, perhaps we can do so here as well?

Reply:
I removed this sentence, good suggestion! I was thinking the same but didn't want to change anything that's unrelated to the new restructuring, so I had kept it.